# HG changeset patch
# User jwe
# Date 797529422 0
#      Mon Apr 10 15:57:02 1995 +0000
# Node ID 3c0e26a1387b9b11f6115a8a8da811962d1ad806
# Parent  9d78b568740065b13912fa2c9126593a86397e86
[project @ 1995-04-10 15:56:53 by jwe]
Initial revision

diff --git a/libcruft/configure.in b/libcruft/configure.in
new file mode 100644
--- /dev/null
+++ b/libcruft/configure.in
@@ -0,0 +1,38 @@
+dnl configure.in
+dnl
+dnl Process this file with autoconf to produce a configure script.
+dnl
+dnl Copyright (C) 1995 John W. Eaton
+### 
+### This file is part of Octave.
+### 
+### Octave is free software; you can redistribute it and/or modify it
+### under the terms of the GNU General Public License as published by the
+### Free Software Foundation; either version 2, or (at your option) any
+### later version.
+### 
+### Octave is distributed in the hope that it will be useful, but WITHOUT
+### ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+### FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+### for more details.
+### 
+### You should have received a copy of the GNU General Public License
+### along with Octave; see the file COPYING.  If not, write to the Free
+### Software Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
+
+AC_REVISION()
+AC_PREREQ(2.0)
+AC_INIT(misc/xstopx.f)
+
+AC_PROG_INSTALL
+
+define([tmpA], [Makefile Makerules])dnl
+define([tmpB], [blas/Makefile balgen/Makefile dassl/Makefile])dnl
+define([tmpC], [eispack/Makefile fftpack/Makefile fsqp/Makefile])dnl
+define([tmpD], [lapack/Makefile linpack/Makefile minpack/Makefile])dnl
+define([tmpE], [misc/Makefile npsol/Makefile odepack/Makefile])dnl
+define([tmpF], [qpsol/Makefile quadpack/Makefile ranlib/Makefile])dnl
+define([tmpG], [slatec-fn/Makefile villad/Makefile])dnl
+define([cruftdirs], [tmpA tmpB tmpC tmpD tmpE tmpF tmpG])dnl
+
+AC_OUTPUT([cruftdirs])
diff --git a/scripts/configure.in b/scripts/configure.in
new file mode 100644
--- /dev/null
+++ b/scripts/configure.in
@@ -0,0 +1,38 @@
+dnl configure.in
+dnl
+dnl Process this file with autoconf to produce a configure script.
+dnl
+dnl Copyright (C) 1995 John W. Eaton
+### 
+### This file is part of Octave.
+### 
+### Octave is free software; you can redistribute it and/or modify it
+### under the terms of the GNU General Public License as published by the
+### Free Software Foundation; either version 2, or (at your option) any
+### later version.
+### 
+### Octave is distributed in the hope that it will be useful, but WITHOUT
+### ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+### FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+### for more details.
+### 
+### You should have received a copy of the GNU General Public License
+### along with Octave; see the file COPYING.  If not, write to the Free
+### Software Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
+
+AC_REVISION()
+AC_PREREQ(2.0)
+AC_INIT(startup/octaverc)
+
+AC_PROG_INSTALL
+
+define([tmpA], [Makefile control/Makefile elfun/Makefile])dnl
+define([tmpB], [general/Makefile image/Makefile])dnl
+define([tmpC], [linear-algebra/Makefile miscellaneous/Makefile])dnl
+define([tmpD], [plot/Makefile polynomial/Makefile set/Makefile])dnl
+define([tmpE], [signal/Makefile specfun/Makefile])dnl
+define([tmpF], [special-matrix/Makefile startup/Makefile])dnl
+define([tmpG], [statistics/Makefile strings/Makefile time/Makefile])dnl
+define([scriptdirs], [tmpA tmpB tmpC tmpD tmpE tmpF tmpG])dnl
+
+AC_OUTPUT([scriptdirs])
